utilities/cli/holohub.py

```diff
@@ -732,7 +732,7 @@ def handle_build(self, args: argparse.Namespace) -> None:
                 build_cmd += " --benchmark"
             if getattr(args, "configure_args", None):
                 for configure_arg in args.configure_args:
-                    build_cmd += f' --configure-args "{configure_arg}"'
+                    build_cmd += f" --configure-args='{configure_arg}'"
```

Comment:
```suggestion
                    build_cmd += f" --configure-args={shlex.quote(args.run_args)}"
```

could you please also help update the other cases
https://github.com/nvidia-holoscan/holohub/blob/d77ef04925a93ab41a8430593d04a247ea2183e4/utilities/cli/holohub.py#L944
https://github.com/nvidia-holoscan/holohub/blob/d77ef04925a93ab41a8430593d04a247ea2183e4/utilities/cli/holohub.py#L1384

Reply:
Updated, thanks!